contracts/utils/RSKAddrValidator.sol

```diff
@@ -0,0 +1,29 @@
+/* solhint-disable no-inline-assembly */
```

Comment:
Solution is too expensive.
The check must be as simple as the following snippet:

   function safeEquals(address addr1, address addr2) internal pure returns (bool){
        return (addr1 == addr2 &&
        addr1 != 0xdcc703c0e500b653ca82273b7bfad8045d85a470&&
        addr1 != address(0));
    }

Reply:
also notice, the address must be not checksumed